packages/map-template/src/components/Map/Map.jsx

```diff
@@ -98,6 +102,20 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
         }
     }, [filteredLocations, filteredLocationsByExternalIDs, mapsIndoorsInstance]);
 
+    /*
+     * React to changes in bearing and pitch props and set them on the map if mapsIndoorsInstance exists.
+     */
+    useEffect(() => {
+        if (mapsIndoorsInstance) {
+            if (pitch && !isNaN(parseInt(pitch))) {
```

Comment:
A more strict check could be used instead of checking if `pitch` is "truthy".
If pitch is set to 0 and it previously was eg. 60, then this will not resolve to true. 

(the same applies to bearing).

Reply:
I read a bit more on this, and saw that it would be better practice to use `Number.isNaN` instead of `isNaN`. Do you think that could be enough? Or did I misunderstand your suggestion?

Reply:
As long as setting pitch to 0 will still trigger the call to tilt(), then do whatever works the best 😄

Reply:
Changed the code to use `Number.isNaN` instead, and it seems to be calling the function correctly

Reply:
But if `pitch` is 0, it will never reach `!Number.isNaN(parseInt(pitch))`, so it will never call `tilt()`. 

It is still making a loose evaluation against "truthyness" for the `pitch` variable.

Reply:
The problem is that the first part of the condition (`pitch`) will return false if the value is the number 0. 

We should not use that check. It's too "loose". And we should make sure that the condition as such returns what we want no matter if the value is `0`, `"0"`, `10`, `"10"`, `null`, `undefined`, `""`, `"aa"`, `[]`, `[0]`, `{}`, `{a: undefined}` etc.

Reply:
Refactored the conditions and the branch is now ready to be reviewed again.